script/generate-signaling-protocol

```diff
@@ -7,23 +7,22 @@ def verbose command
   system(command) || fail("Failed: #{command}")
 end
 
-['SignalingProtocol', 'ScreenSignalingProtocol'].each do |protocol|
```

Comment:
nit: I would just keep the array with a single element in case we need to add other protocol files in the future. But no strong preference.